fixtures/pages-workerjs-wasm-app/package.json

```diff
@@ -5,8 +5,8 @@
 	"sideEffects": false,
 	"scripts": {
 		"check:type": "tsc",
-		"dev": "npx wrangler pages dev public --port 8776",
-		"publish": "npx wrangler pages publish public --experimental-worker-bundle=true",
+		"dev": "npx wrangler pages dev public --bundle --port 8776",
```

Comment:
why do we need `--bundle` here? wasn't that enabled by default?

Reply:
On this commit, it's necessary, since this removes `--experimental-worker-bundle` but doesn't yet default to `-bundle`-ing. I remove it here: https://github.com/cloudflare/workers-sdk/pull/2806/commits/3cc08154d8bf3918e0978dd57c5115c6ed76923c#diff-2df8b4d4d2803bcaf0ed78a19d4f215356b102f8ac581e8f22ca8e0e039d727bR8-R9

Sorry for the confusion!